definitions/variable/technology/electricity-grid.yaml

```diff
@@ -5,7 +5,7 @@
     unit: km
 - Network|Electricity|Expansion Cost:
     description: Cost for increasing the line capacity
-    unit: million EUR/GW/km
+    unit: million EUR/GW/km, EUR_2020/MW
```

Comment:
```suggestion
    unit: [million EUR/GW/km, EUR_2020/MW]
```

Reply:
Not sure how this might affect aggregation if we have two variables of the same name but different units. @sandrinecharousset would you report using both units?

Reply:
*pyam* treats data for the same variable with different units as distinct timeseries, so should aggregate them separately.

Reply:
fine for me, many thanks